packages/app-store/routing-forms/components/react-awesome-query-builder/config/config.tsx

```diff
@@ -29,128 +32,205 @@ const renderComponent = function <T1>(props: T1 | undefined, Component: React.FC
-operators.less.label = "Less than";
-operators.not_equal.label = operators.select_not_equals.label = "Does not equal";
-operators.between.label = "Between";
+function getOperators(configFor: ConfigFor) {
+  // Clone to avoid mutating the original object
+  const operators: Operators = {
+    ...BasicConfig.operators,
+    // Attributes don't need reporting at the moment. So, we can support contains and not contains operators for attributes.
+    ...(configFor === ConfigFor.Attributes
+      ? {
+          multiselect_contains: {
+            label: "Contains",
+            labelForFormat: "CONTAINS",
+            reversedOp: "multiselect_not_contains",
+            // jsonLogic2: "some-in",
+            jsonLogic: function (e, t, r) {
+              return {
+                some: [
+                  e,
+                  {
+                    in: [
+                      {
+                        var: "",
+                      },
+                      r,
+                    ],
+                  },
+                ],
+              };
+            },
+          },
+          multiselect_not_contains: {
+            isNotOp: !0,
+            label: "Not contains",
+            labelForFormat: "NOT CONTAINS",
+            reversedOp: "multiselect_contains",
+            // jsonLogic2: "!some-in",
+            jsonLogic: function (e, t, r) {
+              return {
+                "!": {
+                  some: [
+                    e,
+                    {
+                      in: [
+                        {
+                          var: "",
+                        },
+                        r,
+                      ],
+                    },
+                  ],
+                },
+              };
+            },
+            _jsonLogicIsExclamationOp: !0,
+          },
+        }
+      : {}),
+  };
 
-delete operators.proximity;
-delete operators.is_null;
-delete operators.is_not_null;
+  return operators;
+}
 
-/**
- * Not supported with JSONLogic. Implement them and add these back -> https://github.com/jwadhams/json-logic-js/issues/81
- */
-delete operators.starts_with;
-delete operators.ends_with;
+function getConjunctions(_configFor: ConfigFor) {
+  return {
+    ...BasicConfig.conjunctions,
+  };
+}
 
-const config = {
-  conjunctions: BasicConfig.conjunctions,
-  operators,
-  types,
-  widgets,
-  settings,
+export const FormFieldsBaseConfig = {
+  conjunctions: getConjunctions(ConfigFor.FormFields),
+  operators: getOperators(ConfigFor.FormFields),
+  types: getTypes(ConfigFor.FormFields),
+  widgets: getWidgets(ConfigFor.FormFields),
+  settings: getSettings(ConfigFor.FormFields),
 };
-export default config;
+
+export const AttributesBaseConfig = {
+  conjunctions: getConjunctions(ConfigFor.Attributes),
+  operators: getOperators(ConfigFor.Attributes),
+  types: getTypes(ConfigFor.Attributes),
+  widgets: getWidgets(ConfigFor.Attributes),
+  settings: getSettings(ConfigFor.Attributes),
+};
```

Comment:
Added the structure to ensure that different operators can be returned for FormField and Attributes.